src/libs/CardUtils.ts

```diff
@@ -135,15 +135,26 @@ function maskCard(lastFour = ''): string {
  * Converts given 'X' to '•' for the entire card string.
  *
  * @param cardName - card name with XXXX in the middle.
+ * @param feed - card feed.
  * @returns - The masked card string.
  */
-function maskCardNumber(cardName: string): string {
-    if (!cardName || cardName === '') {
+function maskCardNumber(cardName: string, feed: string | undefined): string {
+    if (!cardName || cardName === '' || !feed) {
```

Comment:
should we leave the feed as potentially undefined? because if it's undefined, this will be true.

Reply:
Updated!